app/Repositories/PublicationsRepository.php

```diff
@@ -13,39 +13,39 @@ public function searchApi($string, $perPage = null, $page = null, $columns = [])
         $search = Search::query()->search($string)->published()->resources(['digital-catalogs', 'printed-catalogs']);
         $results = $search->getSearch($perPage, $columns, null, $page);
 
-        // Now find matching `sections` and add them to the catalogs
-        $searchSections = Search::query()->search($string)->resources(['sections', 'digital-publication-sections']);
-        $resultsSections = $searchSections->getSearch(50, $columns, null, $page);
+        // Now find matching `articles` and add them to the catalogs
+        $searchArticles = Search::query()->search($string)->resources(['articles', 'digital-publication-articles']);
```

Comment:
Keep the reference to `'sections'` the same, as that Data Hub API endpoint will not be renamed. So the array should read:

```
['sections', 'digital-publication-articles']
```